docs/design/Non_Admin_Controller_design.md

```diff
@@ -0,0 +1,209 @@
+# Non-Admin Backup/Restore Design
+
+## Background
+[OADP (Openshift API for Data Protection)](https://github.com/openshift/oadp-operator) Operator currently requires cluster admin access for performing Backup and Restore operations of applications deployed on the OpenShift platform.
+This design intends to enable non-admin users the ability to perform Backup and Restore operations of their application namespace.
+
+## Goals
+- Enable non-admin backup operation
+- Enable non-admin restore operation
+
+## Non-Goals
+- Performance improvements of Backup and Restore Operations
+- Parallel Backup and Restore Operations
+
+## Use-Cases
+
+### Backup Operation
+- As a non-admin user/namespace owner with administrative privileges for a particular namespace, the user should be able to:
+    - Create a Backup of the namespace
+    - View the status of the Backup created for the particular namespace
+    - Delete the Backup of the namespace
+
+### Restore Operation
+- As a non-admin user/namespace owner with administrative privileges for a particular namespace, the user should be able to:
+    - Create a Restore of the namespace
+    - View the status of the Restore created for the particular namespace
+    - Delete the Restore of the namespace
+
+### BackupStorageLocation (BSL) Configuration Operation (optional use case)
```

Comment:
Should we mention the use case of that BSL in the other operations such as Backup? How this will be used?

Reply:
added the use-case